flink-libraries/flink-table/src/main/scala/org/apache/flink/table/runtime/aggregate/GroupAggProcessFunction.scala

```diff
@@ -162,7 +162,7 @@ class GroupAggProcessFunction(
 
   override def onTimer(
       timestamp: Long,
-      ctx: ProcessFunction[CRow, CRow]#OnTimerContext,
+      ctx: ProcessFunction[CRow, CRow]#OnTimerContext[_],
```

Comment:
I'm not a scala expert, but is this change somehow related to adding the `getCurrentKey()` method?

Reply:
yes, I think I'll create a `KeyedProcessFunction` as Aljoscha suggested, and we probably don't need these scala changes any more.